yaboop/todo.js

```diff
@@ -0,0 +1,92 @@
+const STATUS = {
+  TODO: "To Do",
+  IN_PROGRESS: "In Progress",
+  DONE: "Done"
+}
+
+const list = {
+	"create a new practice task": STATUS.IN_PROGRESS, 
+	"make a bed": STATUS.DONE, 
+	"write a post": STATUS.TODO,
+  "play a game": STATUS.TODO
+}
+
+const ERRORS = {
+  NOT_FOUND: "Задача не найдена, введите задачу",
+  NOT_EXIST: ": Задача не существует"
+}
+
+function changeStatus (task, status) {
+  if (task === "" || status ==="") {
+    console.log (task + ERRORS.NOT_FOUND)
+  } else if (list[task]) {
+    list[task] = status;
+} else console.log(task + ERRORS.NOT_EXIST);
+}
+
+function addTask (task) {
+  if (task === "") {
+    console.log (task + ERRORS.NOT_FOUND)
+  } else list[task] = "To Do" 
+}
+
+function deleteTask (task) {
+  if (task === "") {
+    console.log (task + ERRORS.NOT_FOUND)
+  } else if (list [task]) {
+    delete list[task];
+  } else console.log(task + ERRORS.NOT_EXIST);
+}
+
+
+function showList() {
+
+  let counter = 0;
+
+  console.log("To Do:");
```

Comment:
Статусы можно брать из переменных
```suggestion
  console.log(STATUS.TODO, ":");
```